ccd-definition/AuthorisationCaseField/CareSupervision/system-update.json

```diff
@@ -2770,7 +2770,7 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseFieldID": "gatekeepingOrderSealDecision",
     "UserRole": "caseworker-publiclaw-systemupdate",
-    "CRUD": "CRU"
+    "CRUD": "CU"
```

Comment:
Just wondering if this might cause any issues if the system update user can't read?

Reply:
I'll add it back as it is not a real user.